actors/cron/src/lib.rs

```diff
@@ -56,7 +58,12 @@ impl Actor {
         let st: State = rt.state()?;
         for entry in st.entries {
             // Intentionally ignore any error when calling cron methods
-            let res = rt.send(&entry.receiver, entry.method_num, None, TokenAmount::zero());
+            let res = extract_send_result(rt.send_simple(
```

Comment:
This looks really painful. It would be simpler to:

1. Introduce a new `SendError(ErrorNumber)` type.
2. Implement `From<SendError> for ActorError` for easy conversion.

Reply:
Nvm. This is going to be harder due to the fact that:

1. Types live in the wrong crates.
2. We have two layers of errors.